tests/profile.yaml

```
@@ -0,0 +1,18 @@
+
+# verify data is visible 
+- assertVisible: "Ime"
+- assertVisible: "Prezime"
+- assertVisible: "OIB"
+- assertVisible: "E-mail"
+
+# test accuracy
+- assertVisible: "Test3"
+- assertVisible: "Osoba3"
+- assertVisible: "02535077085"
+- assertVisible: "osoba3@test.hr" 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)